event-log-core/src/main/scala/com/ubirch/services/kafka/consumer/DefaultStringConsumer.scala

```diff
@@ -163,6 +164,8 @@ class DefaultStringConsumer @Inject() (
   with ConsumerCreator
   with WithConsumerShutdownHook {
 
+  private implicit val scheduler: Scheduler = Scheduler(ec)
```

Comment:
I changed here little bit more. I injected `Scheduler` instead of `ExecutionContext` .

Kafka-express is using `Scheduler` now, which might improve the performance. But I don't see a big affect.